apps/judicial-system/web/src/components/Logo/Logo.tsx

```diff
@@ -11,14 +11,21 @@ interface Props {
 
 const Logo: React.FC<Props> = ({ defaultInstitution = '' }) => {
   const { user } = useContext(UserContext)
+  const institutionName = user?.institution?.name ?? defaultInstitution
+  const institutionNameArr = institutionName.split(' ')
+  const institutionNameFirstHalf = institutionNameArr.slice(
+    0,
+    institutionNameArr.length - 1,
+  )
 
   return (
     <div className={styles.logoContainer}>
       <Box marginRight={[0, 0, 0, 2]} marginBottom={[0, 0, 1, 0]}>
         <LandWightsLogo />
       </Box>
       <p className={styles.logoText}>
```

Comment:
Should this be extracted into a separate component and that component tested, instead of mounting the entire Cases page to test this, like we are doing in this pr? What do you think?

Reply:
This component is so small. This is basically the only logic in it. I think either testing this component or you can extract this logic to a function and test that.